packages/plans-grid-next/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/plans-grid-next",
-	"version": "1.0.0",
+	"version": "1.0.1",
```

Comment:
Why are we bumping those versions?

Reply:
Linking to a previous comment :)
https://github.com/Automattic/wp-calypso/pull/101952#discussion_r2023235321

Reply:
I can see we're using this package directly in wpcom, so that makes sense 👍